cli/BUILD.gn

```diff
@@ -217,6 +217,7 @@ run_node("deno_runtime_declaration") {
              "//tools/ts_library_builder/build_library.ts",
              "//tools/ts_library_builder/ast_util.ts",
            ]
+  data = inputs
```

Comment:
I don't understand why this is needed.

Reply:
So that `gn desc target/release cli:deno runtime_deps` returns a list of TS files we depend on E.G. `js/main.ts`

Reply:
This is definitely not idiomatic use of the gn `data` property.
Also `./buildtools/mac/gn desc target/release cli:deno_runtime_declaration` does list these files without this hack.
So it makes me suspicious there are other indirect action targets where sources / inputs are not properly picked up properly.

Reply:
The label `cli:deno_runtime_declaration` is private as far as cargo is concerned. The top level target is `cli:deno_deps` and I'm not able to see these files in `gn desc target/release cli:deno_deps`...

Reply:
```
Gallivant:deno piscisaureus$ touch js/dir.ts 
Gallivant:deno piscisaureus$ tools/build.py 
ninja: Entering directory `/Users/piscisaureus/d/deno/target/debug'
[18/18] STAMP obj/default.stamp
```

So somehow ninja does know about these deps.

Reply:
`third_party/depot_tools/ninja -C target/release/ -t query gen/cli/lib/lib.deno_runtime.d.ts` lists them